cases/views.py

```diff
@@ -412,15 +412,15 @@ def show_confirmation_step(wizard):
 
 
 def compile_dates(data):
-    start = datetime.datetime.combine(
-        data["start_date"], data["start_time"], tzinfo=timezone.get_current_timezone()
-    )
+    tz = timezone.get_current_timezone()
+    start = datetime.datetime.combine(data["start_date"], data["start_time"])
+    start = tz.localize(start)
```

Comment:
Sorry, you probably didn't notice my later message in slack. I think (worth checking!) that this is equivalent:
```
start = datetime.datetime.combine(data["start_date"], data["start_time"])
start = timezone.make_aware(start)
```

Reply:
Ah nice